### DISCUSSION TITLE
【灌水】What can I say？

### DISCUSSION THREAD

**Discussion by keerdun** [share121/inter-knot#1320] [其他]
![IMG_20240811_132741](https://github.com/user-attachments/assets/66a06725-6b5b-4a52-a6d3-9eeba760b7ac)
 不懂就问，治安局的宣传都这样嘛

**Comment by su2569** [comment]
离谱

**Comment by ZhuYuan-148** [comment]
前辈好帅❤

**Reply by Qiling-XwX** [reply]
警官我就抱走了

**Comment by T7moris** [comment]
楼上是朱鸢警官小号吗，怎么又实名上网
